unity-renderer/Assets/Scripts/MainScripts/DCL/AvatarSystem/BaseAvatar.cs

```diff
@@ -54,21 +53,22 @@ public async UniTask FadeOut(MeshRenderer targetRenderer, bool playParticles, Ca
             if (avatarRevealerContainer == null) 
                 return;
             
-            disposeCts ??= new CancellationTokenSource();
-            CancellationToken linkedCt = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken, disposeCts.Token).Token;
+            transitionCts ??= new CancellationTokenSource();
+            CancellationToken linkedCt = CancellationTokenSource.CreateLinkedTokenSource(cancellationToken, transitionCts.Token).Token;
             linkedCt.ThrowIfCancellationRequested();
             
             avatarRevealer.AddTarget(targetRenderer);
-            await avatarRevealer.StartAvatarRevealAnimation(playParticles && !isInsideHideArea, linkedCt);
+            await avatarRevealer.StartAvatarRevealAnimation(playParticles, linkedCt);
```

Comment:
Let's rename `playParticles` to `withTransition` (or `instant` and negate it)

Reply:
Let's also add a comment to clarify that `StartAvatarRevealAnimation`  completes the transition when canceled.